_zip/missingapi.yml

```diff
@@ -29,15 +29,11 @@ references:
   name: PolicyRegistry
   fullname: Polly.Registry.PolicyRegistry
   href: https://github.com/App-vNext/Polly/wiki
-- uid: Polly.Registry.IPolicyRegistry%601
+- uid: Polly.Registry.IPolicyRegistry`1
   name: IPolicyRegistry<TKey>
   fullname: Polly.Registry.IPolicyRegistry<TKey>
   href: https://github.com/App-vNext/Polly/wiki
-- uid: Polly.Registry.IReadOnlyPolicyRegistry%601
-  name: IReadOnlyPolicyRegistry<TKey>
-  fullname: Polly.Registry.IReadOnlyPolicyRegistry<TKey>
-  href: https://github.com/App-vNext/Polly/wiki
-- uid: Polly.Registry.IReadOnlyPolicyRegistry%601
+- uid: Polly.Registry.IReadOnlyPolicyRegistry%`1
```

Comment:
Looks like there's still an extra `%` here :)

Reply:
Doh!

Reply:
Looks like that did the trick ... there's two more that are still showing missing xref warnings. If you add them, this should get wrapped up :)